pypika/tests/test_query.py

```diff
@@ -120,6 +115,20 @@ def test_replace_orderby_table(self):
             'SELECT "customer" ' 'FROM "b" ' 'ORDER BY "customer"', str(query)
         )
 
+    def test_replace_tuple_table(self):
+        query = (
+            Query.from_(self.table_a)
+            .select(self.table_a.cost, self.table_a.revenue)
+            .where(Tuple(self.table_a.cost, self.table_a.revenue) == Tuple(1, 2))
```

Comment:
You shouldn't need to use the Tuple class directly. You can just use a python tuple. Pypika wraps python tuples with it's own tuple internally.

Reply:
I've updated it to use a python tuple instead.